docs/automation/README.md

```diff
@@ -46,10 +68,11 @@ curl
 
 
 # To apply new password
+# token - user will get by email if you have Integration for SMTP
 curl 
 -X POST -d '{
   "new_password": "jexia_super"
-}' "https://$PROJECT_ID.app.jexia.com/ums/resetpassword/token" | jq .
+}' "https://$PROJECT_ID.app.jexia.com/ums/resetpassword/"+token | jq .
```

Comment:
Are you sure that `+token` will work? Bash it's not programming language and you cannot just add the string like this

Reply:
put it back